pkg/cloudprovider/providers/openstack/openstack.go

```diff
@@ -100,13 +100,18 @@ type RouterOpts struct {
 	RouterId string `gcfg:"router-id"` // required
 }
 
+type MetadataOpts struct {
+	SearchOrder []string `gcfg:"search-order"`
```

Comment:
Is this('[]string') easy to use for user?

Reply:
`[]string` would be the standard for ini config types. But we can also use a comma separated value. Which do you think it's more user friendly?

Reply:
It is better to use a comma separated.